sel/sel.go

```diff
@@ -13,47 +15,46 @@ func AllowAllFilter(string, string) bool {
 }
 
 func MakeFilter(include, exclude []string) NSFilter {
+	lg := log.New("filter")
+
 	if len(include) == 0 && len(exclude) == 0 {
 		return AllowAllFilter
 	}
 
 	includeFilter := doMakeFitler(include)
```

Comment:
nit, typo: `Fitler` -> `Filter`

Just a nit since it's not introduced by the PR.